tests/e2e/utils/flows.js

```diff
@@ -589,43 +587,6 @@ export const merchantWCP = {
 		} );
 	},
 
-	openDisputeDetails: async ( disputeDetailsLink ) => {
```

Comment:
This helper function is no longer relevant.

Reply:
Nice. I mentioned it part of the https://github.com/Automattic/woocommerce-payments/issues/7363's tasks.